SC_redesign/Crop_and_Soil/crop/crop_data.py

```diff
@@ -204,20 +266,11 @@ class CropData:
     
     # ---- yields
     # constant attributes
-    optimal_harvest_index: float = 3.5
-    """expected species-specific optimal harvest index for the plant at maturity under ideal
-     growth conditions (unitless)"""
-    min_harvest_index: float = 0.2
-    """expected species-specific harvest index for the plant under drought conditions; represents minimum harvest index
-    allowed for the plant (unitless)"""
+
     # is_residue_added: bool = False ## not needed?
     harvest_efficiency: float = 1.0
     """efficiency of the harvest operation: the proportion of yield that will be extracted from the field 
     (unitless; [0, 1])"""
```

Comment:
```suggestion
    (unitless; must be greater than or equal to 0 and less than or equal to 1)"""
```
Not really a necessary change, but it would eliminate the 30 seconds I needed to think about what `[0, 1]` meant

Reply:
This is a good point - I have been using set notation occasionally. I'll need to think about this more.